app/views/index.html

```diff
@@ -46,6 +46,7 @@
             window.__ROOM_NAME__ = "${roomName}";
             window.__USER_ADMIN__ = "${user.isAdmin()}";
             window.__USER_GUEST__ = "${user.isGuest()}";
+            document.title="${room_name"} + " : breaker";
```

Comment:
Thanks, I hadn't pushed it though. It has a typo too , a misplaced 
quotation mark  after room_name.

On 29-Aug-16 16:14:37, "Matt Hall" <notifications@github.com> wrote:

>In app/views/index.html:
>
> > @@ -46,6 +46,7 @@ > window.__ROOM_NAME__ = "${roomName}"; > 
>window.__USER_ADMIN__ = "${user.isAdmin()}"; > window.__USER_GUEST__ = 
>"${user.isGuest()}"; > + document.title="${room_name"} + " : breaker";
>I removed this line after merging BTW. Wasn't compiling.
>
>—
>You are receiving this because you authored the thread.
>Reply to this email directly, view it on GitHub, or mute the thread.
>